doc/verification.rst

```diff
@@ -0,0 +1,68 @@
+.. _Verification:
+
+Verification and Validation
+===========================
+This guide is inteded to be used to document major ``ADflow`` cases for validation and verification purposes.
+
+
+Overset and Multiblock Meshes
+-----------------------------
+The `following paper <https://arc.aiaa.org/doi/pdf/10.2514/1.C034486>`_ compares ``ADflow`` with OVERFLOW and elsA for the CRM and a Wing-Body-Nacelle-Pylon configuration. 
+Both overset and multiblock structured meshes were used in the comparison. The cases are run at transonic mach numbers and Reynolds Numbers of ``Re = 5e6``. Figure 8 of in this paper details drag grid convergence of each of the flow solvers.
+
+.. code-block:: bibtex
+
+    @incollection{Coder2017a,
+        author = {Coder, James G. and Pulliam, Thomas H. and Hue, David and Kenway, Gaetan K. W. and Sclafani, Anthony J.},
+        booktitle = {AIAA SciTech Forum},
+        doi = {10.2514/6.2017-0960},
+        month = {January},
+        publisher = {American Institute of Aeronautics and Astronautics},
+        title = {Contributions to the 6th {AIAA} {CFD} {Drag Prediction Workshop} Using Structured Grid Methods},
+        year = {2017}
+    }
+
+Hydrofoils
+----------
+The following `paper <https://www.sciencedirect.com/science/article/abs/pii/S088997461630144X?via%3Dihub>`_ couples hydrodynamic and structural analysis of hydrofoils and compares the resutls with experimental data.
```

Comment:
same as above. have an identifier in the text so that people know you are talking about the paper below. also change the URL so that it redirects from doi.org

Reply:
Same here